ambry-protocol/src/main/java/com.github.ambry.protocol/PartitionResponseInfo.java

```diff
@@ -104,6 +104,9 @@ public String toString() {
     sb.append("PartitionId=").append(partitionId);
     sb.append(" ServerErrorCode=").append(errorCode);
     sb.append(" MessageInfoAndMetadataListSize=").append(messageInfoAndMetadataListSize);
+    if (errorCode == ServerErrorCode.No_Error) {
```

Comment:
`MessageInfoList` is relevant enough to go in the public access log and is already being logged for `ReplicaMetadataResponse`. Doing the same for Gets here. The downside is that this will bloat the public access log, so we can make a call later on whether this needs to be optimized away (and only relevant information is kept).

Reply:
Not sure if we should get rid of this. It can be large, but can be useful so this is a tradeoff between how much we want for a request-response vs. how long back in the past we get to go. 

I will make the change in a separate PR while we get to a consensus on this.